charts/nextcloud/values.yaml

```diff
@@ -675,12 +683,50 @@ tolerations: []
 
 affinity: {}
 
-dnsConfig: {}
-# Custom dns config for Nextcloud containers.
-# You can for example configure ndots. This may be needed in some clusters with alpine images.
-# options:
-#   - name: ndots
-#     value: "1"
+dnsConfig:
+  # -- Custom dns config for Nextcloud containers.
+  # You can for example configure ndots. This may be needed in some clusters with alpine images.
+  # options:
+  #   - name: ndots
+  #     value: "1"
+  options: []
+
+
+# Notify Push (Clientpush)
+notifyPush:
+  # -- enable another deployment to handle notify_push (sometimes called ClientPush)
+  enabled: false
+  autoSetup: false
+
+  replicaCount: 1
+
+  image:
+    registry: docker.io
+    # -- image of notify_push (there is no official image yet: https://github.com/nextcloud/notify_push/issues/106)
+    repository: miles170/notify_push
```

Comment:
This doesn't feel good to me, anything that is not at least semi-official (like the nextcloud docker image) is a risky dependency to add.

Reply:
Could you try the AIO container? I'd be more comfortable with that, but if it doesn't work we can use this image for now.